vsphere/datadog_checks/vsphere/data/conf.yaml.example

```diff
@@ -71,6 +21,33 @@ instances:
     #
     password: <PASSWORD>
 
+    ## @param collection_level - integer - optional - default: 1
+    ## A number between 1 and 4 to specify how many metrics will be sent
+    ## 1: Only basic metrics - 4: every metric available.
+    ## Warning: Depending on the size of the vSphere environment, metric collection can be slow,
+    ## very CPU intensive and put pressure on the vCenter Server
+    ## If you want more control on the metrics to collect, see `metric_filters`
+    #
+    # collection_level: 1
```

Comment:
Can we document what options `2` and `3` will also collect?

Reply:
Added a link 👍